docker/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "Waiting for database..."
+while ! python -c "import socket; s = socket.socket(); s.connect(('db', 5432))"; do
+  sleep 1
+done
+echo "Database is ready."
+
+python forum/manage.py makemigrations
+python forum/manage.py migrate --fake-initial
+python forum/manage.py collectstatic --noinput
+
+gunicorn forum.forum.wsgi:application --bind 0.0.0.0:8000 --workers 3 --timeout 120
```

Comment:
The Gunicorn configuration is hardcoded, which may not work well for all environments. Allow customization via environment variables:
```
: "${GUNICORN_WORKERS:=3}"
: "${GUNICORN_TIMEOUT:=120}"
gunicorn forum.wsgi:application --bind 0.0.0.0:8000 --workers "$GUNICORN_WORKERS" --timeout "$GUNICORN_TIMEOUT"
```